### DISCUSSION TITLE
iptables to restrict LAN access

### DISCUSSION THREAD

**Discussion by Jim-Young** [wg-easy/wg-easy#1286] [General]
Hi hope someone can explain why I have to add "iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE"  to make it work:

WG_POST_UP=iptables -I FORWARD -i wg0 -d 192.168.6.0/24 -j REJECT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE

I checked config.js and there is already a default script:
iptables -t nat -A POSTROUTING -s ${module.exports.WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o ${module.exports.WG_DEVICE} -j MASQUERADE;

It won't work after I removed "iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE" 

Thanks

**Comment by DevGoran** [comment]
I believe the following part of code is handling this:

```
module.exports.WG_POST_UP = process.env.WG_POST_UP || `
iptables -t nat -A POSTROUTING -s ${module.exports.WG_DEFAULT_ADDRESS.replace('x', '0')}/24 -o ${module.exports.WG_DEVICE} -j MASQUERADE;
iptables -A INPUT -p udp -m udp --dport ${module.exports.WG_PORT} -j ACCEPT;
iptables -A FORWARD -i wg0 -j ACCEPT;
iptables -A FORWARD -o wg0 -j ACCEPT;
`.split('\n').join(' ');
```

From what I understand is it will either use the values from the environment values or these default values. I believe the || is a logical OR.
